### PR TITLE
[WIP] misc: adds the description in each file

### PR DESCRIPTION
**Pull request by pranshuchittora**
It's hard to understand what each file in the code does. By adding a bit of description at the top in each file can be quite helpful.
```
Name : <FILE NAME>
Description : <Info about the core functionality>
```

**What kind of change does this PR introduce?**
Improves DX

### CODE REVIEW

**Comment by pranshuchittora** [comment]
As of now, I have added the description to only one file. 
Let's discuss its pros & cons. :fire:

File: packages/generators/init-generator.ts [hunk 1]
```diff
@@ -1,3 +1,8 @@
+/*
+Name 			: init-generators
+Description 	: Generates new webapck configuration as per user requirements.
```
Comment (evenstensberg) [inline packages/generators/init-generator.ts]: spelling

**Comment by ematipico** [comment]
@pranshuchittora you could mark this PR as Draft PR, if you think this is not ready. If you can't mark it as "Draft", could you add the "[WIP]" inside the title?

File: packages/generators/init-generator.ts [hunk 2]
```diff
@@ -1,3 +1,8 @@
+/*
```
Comment (rishabh3112) [inline packages/generators/init-generator.ts]:
```suggestion
/**
```

**Comment by evenstensberg** [comment]
Hi. As long as we have descriptive code and JSDocs (instead of comments on top), I think this is redundant. Thanks for putting in the time!

**Pull request closed without merging**